package.json

```diff
@@ -76,16 +75,19 @@
     "node": ">=18.0.0"
   },
   "scripts": {
+    "dev": "echo \"Use 'dev:16', 'dev:18', or 'dev:19' depending on the version of React you want to test\"",
+    "dev:16": "REACT_VERSION=16 vite",
+    "dev:18": "REACT_VERSION=18 vite",
+    "dev:19": "REACT_VERSION=19 vite",
```

Comment:
I like the new way of testing different react versions. What do you think about making the latest version to be the default?

Reply:
This may decrease visibility of other options.

Those demos have quite unclear audience. I think that mostly us are running them. Other people, to my intuition, probably just scan their code. 

Exposing the ability to test with previous React versions to us would be a nice benefit. It increases the awareness of potential issues.